staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -693,6 +724,29 @@ type PersistentVolumeClaimStatus struct {
 	// +mapType=granular
 	// +optional
 	AllocatedResourceStatuses map[ResourceName]ClaimResourceStatus `json:"allocatedResourceStatuses,omitempty" protobuf:"bytes,7,rep,name=allocatedResourceStatuses"`
+	// volumeAttributesClassName is the name of VolumeAttributesClass which is latest applied by
+	// the CSI driver. This field is not set by default but when modification is complete,
```

Comment:
Clarify what "This field is not set by default but when modification is complete" means and offer any clarification to when the field _is_ set?

Reply:
```go
// This field is not set by default but when modification is complete,
// it's set to the applied VolumeAttributesClass.
```